static/js/components/InstrumentSummary.jsx

```diff
@@ -60,9 +61,7 @@ const InstrumentPlot = ({ instrumentId, startDate, endDate }) => {
         </div>
       }
     >
-      <Plot
-        url={`/api/internal/plot/instrument_log/${instrumentId}?width=${plotWidth}&height=500&startDate=${startDate}&endDate=${endDate}`}
-      />
+      <div>TODO</div>
```

Comment:
Just flagging this here.

Reply:
yikes I forgot about that instruments log plot... will re-implement it too!